### DISCUSSION TITLE
Error of Visualization of trajectory by CellDimPlot

### DISCUSSION THREAD

**Discussion by ZennFangZhou** [pwwang/scplotter#15] [General]
**When i run the code:**

CellDimPlot(pancreas_sub, group_by = "SubCellType", reduction = "UMAP",
            lineages = paste0("Lineage", 1:3), lineages_span = 0.1)

**i received the error:**
  
Error in scale$guide == "none" : 
  comparison (==) is possible only for atomic and list types

the error is dued to the version of dependency of R packages?

thanks!

**Comment by pwwang** [comment]
I can't reproduce this. Can you attach your session info?
